starport/pkg/spn/proposal.go

```diff
@@ -8,13 +8,23 @@ import (
 	"github.com/tendermint/starport/starport/pkg/jsondoc"
 )
 
-// ProposalStatus keeps a proposal's status state.
+// ProposalType represents the type of the proposal
+type ProposalType string
+
+const (
+	ProposalTypeAll          = ""
+	ProposalTypeAddAccount   = "add-account"
+	ProposalTypeAddValidator = "add-validator"
+)
+
+// ProposalStatus represents the status of the proposal
 type ProposalStatus string
 
 const (
-	ProposalPending  = "pending"
-	ProposalApproved = "approved"
-	ProposalRejected = "rejected"
+	ProposalStatusAll      = ""
```

Comment:
```suggestion
	ProposalStatusAll   ProposalStatus   = ""
```

And for others as well.